recipes/temporalio/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "temporalio" %}
+{% set version = "1.14.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/temporalio-{{ version }}.tar.gz
+  sha256: b240cf56f64add65beb75bd18aa854ac35bdc2505097af5af1e235d611190a9d
+
+build:
+  skip: true  # [win and py<39]
```

Comment:
Since our minimum Python version is already 3.9, this won't actually skip any builds, so you may still run out of disk space.  Feel free to skip Windows entirely for now if it still runs out of space, and then just re-enable it in the feedstock.  The nice thing in the feedstock is that each Python version is built in a totally separate job, so you shouldn't have any disk spaces issues at that point.

Reply:
Thanks for spotting this Daniel, that's helpful. My quick impression was that `win and py<39` would be a union of those 2 conditions, but I now expect I need an `or` here.